src/bundle/Resources/public/scss/_popup-menu.scss

```diff
@@ -15,19 +14,48 @@
     &__item {
         display: flex;
         align-items: center;
-        height: calculateRem(35px);
-        padding: calculateRem(20px);
+        height: calculateRem(40px);
         min-width: calculateRem(150px);
+        transition: all $ibexa-admin-transition-duration $ibexa-admin-transition;
+
+        &--hidden {
+            display: none;
+        }
+    }
+
+    a.ibexa-popup-menu__item-action {
```

Comment:
is `a` needed here?

Reply:
yes, because otherwise I can't overwrite the style